conda-recipe/meta.yaml

```diff
@@ -16,7 +16,6 @@ build:
 requirements:
   build:
     - python
-    - six
 
   run:
     - python
```

Comment:
`jinja2` for that matter.  `prettytable` is only used inside of a `try...except` block in `__str__` and gracefully falls back to `dict.__str__` if import fails.  `jinja2` and `humanize` are only imported/used inside of `_repr_html_` and we should sort out how to make sure that fails gracefully or just remove it.